l10n_it_abicab/models/abicab.py

```diff
@@ -17,8 +18,8 @@ class ResPartnerBank(models.Model):
 
     _inherit = "res.partner.bank"
 
-    bank_abi = fields.Char(size=5, string='ABI')
-    bank_cab = fields.Char(size=5, string='CAB')
+    bank_abi = fields.Char(size=5, string='ABI', related='bank_id.abi')
+    bank_cab = fields.Char(size=5, string='CAB', related='bank_id.cab')
```

Comment:
@primes2h since they were DB columns, I would add `store=True`. What do you think?

Reply:
Better :+1: 

Done.